pandas/tseries/frequencies.py

```diff
@@ -144,6 +154,24 @@ def to_offset(freq) -> Optional[DateOffset]:
         except ValueError as err:
             raise ValueError(libfreqs.INVALID_FREQ_ERR_MSG.format(freq)) from err
 
+    elif isinstance(freq, int):
+        stride = 1
+        options = {
```

Comment:
Have there been any discussions with dateutil to expose this as an enum? This seems a little strange to do on the pandas end